File: print.go
package main
import "fmt"
import "os"
import "time"
func main() {
  fmt.Println(os.Getpid())
  for i:=0;i<1000;i++ {time.Sleep(time.Millisecond * 10)}
  fmt.Println(os.Getenv("PATH"))
}

File: print.rs
use std::env;
use std::process;
use std::thread;
use std::time;

fn main() {
  println!("{}",process::id());
  for _i in 0..1000 {thread::sleep(time::Duration::from_millis(10));}
  match env::var("PATH") {
    Ok(val) => println!("{}",val),
    Err(_) => (),
  }
}
